ui/digural/src/main/kotlin/de/cyface/app/digural/capturing/CapturingFragment.kt

```diff
@@ -879,6 +879,7 @@ class CapturingFragment : Fragment(), DataCapturingListener, CameraListener {
         val staticExposureTimeSelected = cameraSettings.getStaticExposureBlocking()
         val staticExposureTime = cameraSettings.getStaticExposureTimeBlocking()
         val exposureValueIso100 = cameraSettings.getStaticExposureValueBlocking()
+        val anonymizationModel = cameraSettings.getAnonModel()
```

Comment:
Digural defaults to Model 1 (see last CameraService commit)

Reply:
TODO: test - the background service should opt to NoAnonymization, but we need to make sure this is visible in the Settings.